library/src/main/java/com/novoda/downloadmanager/NotificationChannelCreator.java

```diff
@@ -0,0 +1,11 @@
+package com.novoda.downloadmanager;
+
+import android.app.NotificationChannel;
+
+public interface NotificationChannelCreator {
+
+    Optional<NotificationChannel> createNotificationChannel();
```

Comment:
If this is gonna be part of the public API of the library can we have some javadoc or something explaining why it's `Optional`?

Reply:
Sure

Reply:
https://novoda.atlassian.net/browse/ALL-4581